pkg/graph/schema.graphql

```diff
@@ -1913,9 +1913,9 @@ enum PayType {
 
 enum ClaimsBasedPayType {
   ADJUSTMENTS_TO_FFS_PAYMENTS
-  PAYMENTS_CARE_MANAGEMENT_HOME_VISITS
-  PAYMENTS_SNF_CLAIMS_WITHOUT_3DAY_HOSPITAL_ADMISSIONS
-  PAYMENTS_TELEHEALTH_SERVICES_NOT_TRADITIONAL_MEDICARE
+  CARE_MANAGEMENT_HOME_VISITS
+  SNF_CLAIMS_WITHOUT_3DAY_HOSPITAL_ADMISSIONS
+  TELEHEALTH_SERVICES_NOT_TRADITIONAL_MEDICARE
   OTHER
```

Comment:
There looks like some fields have gone missing, like `REDUCTIONS_TO_BENEFICIARY_COST_SHARING` and `  SERVICES_NOT_COVERED_THROUGH_TRADITIONAL_MEDICARE`

This was made evident to me through the merge conflicts

Reply:
I see the same thing when looking at the merge conflicts. @OddTomBrooks can you look into this?

Reply:
Yep, it was just a bit off the head from main. Updated now.